source/_integrations/todo.markdown

```diff
@@ -80,12 +80,12 @@ data:
 
 Add a new to-do item. A to-do list `target` is selected with a [Target Selector](/docs/blueprint/selectors/#target-selector) and the `data` payload supports the following fields:
 
-| Data attribute | Optional | Description                                                      | Example                                                      |
-| -------------- | -------- | ---------------------------------------------------------------- | ------------------------------------------------------------ |
-| `item`         | no       | the name of the to-do Item.                                      | Submit income tax return                                     |
-| `due_date`     | yes      | The date the to-do item is expected to be completed.             | 2024-04-10                                                   |
-| `due_datetime` | yes      | The date and time the to-do item is expected to be completed.    | 2024-04-10 23:00:00                                          |
-| `description`  | yes      | A more complete description than the one provided by the summary | Collect all necessary documents and submit the final return. |
+| Data attribute | Optional | Description                                                       | Example                                                      |
+| -------------- | -------- | ----------------------------------------------------------------- | ------------------------------------------------------------ |
+| `item`         | no       | The name and summary of the to-do item.                           | Submit income tax return                                     |
```

Comment:
```suggestion
| `item`         | no       | The name/summary of the to-do item.                                | Submit income tax return                                     |
```
describing it as "name and summary" makes it seem that there are 2 values here. maybe a slash?